ert_shared/models/base_run_model.py

```diff
@@ -81,6 +81,7 @@ def startSimulations(self, arguments):
             self.completed_realizations_mask = run_context.get_mask()
         except ErtRunError as e:
             self.completed_realizations_mask = BoolVector(default_value=False)
+            self.updateDetailedProgress()
             self._failed = True
             self._fail_message = str(e)
```

Comment:
What about in the UserWarning case ? Should we update there too ?

Reply:
Yeah we can do that. I have no idea what UserWarning is.